js/common/trace.go

```diff
@@ -0,0 +1,8 @@
+package common
+
+import "go.opentelemetry.io/otel/trace"
+
+// TracerProvider provides methods for OTEL tracers initialization.
+type TracerProvider interface {
```

Comment:
I was doubtful whether to define this in _[js/modules/modules.go](https://github.com/grafana/k6/blob/9d1d028392094a8de6897f752784e3dd22a8371d/js/modules/modules.go#L39)_ along with the _VU_ interface, but decided to define it in `common` package mimicking the `Events` implementation and also thinking that more tracing related functionalities could be added here in the future. Feedback is welcomed.

Reply:
Having given this a quick thought, I do think it makes sense to have it in common, considering it was explicitly designed to be reusable by different parts/modules of k6. 

I also remember we already discussed doing something along those lines as we implemented the `k6/experimental/tracing` module, so 👍🏻 on my side.

Reply:
Due to the changes discussed in https://github.com/grafana/k6/pull/3445#discussion_r1386243267, the `TracerProvider` interface that defines the methods to be exposed to modules has been [moved](https://github.com/grafana/k6/blob/6225e08ce8e956d534da839f1116f3c8d346f088/lib/vu_state.go#L24) to `lib` package, as otherwise there was an import cycle between `common` and `lib` packages.